sklearnex/dispatcher.py

```diff
@@ -412,6 +413,19 @@ def get_patch_map_core(preview=False):
             ]
         ]
 
+        if daal_check_version((2024, "P", 600)):
```

Comment:
adding IncrementalRidge to patching (not preview) without this check fails public ci, since it only works from onedal version 2024.6.0. there is one test for test_patching that takes estimators from patching and calls their methods, however it fails on backend since public ci uses 2024.5.0